e2e_test/.coverageignore

```diff
@@ -15,3 +15,6 @@ e2e_test/source_legacy/pulsar/**/*.slt*
 
 # Manually executed in `e2e_test/subscription/main.py`
 e2e_test/subscription/**/*.slt*
+
+# Manually executed in `run-e2e-tests.sh`
+e2e_test/background_ddl/**/*.slt*
```

Comment:
I think it's not executed at all

Reply:
Could you add a line to execute it inside `e2e-test-serial.sh`?

You can ctrl-F `background_ddl`. There's a line like:
```
risedev slt -p 4566 -d dev './e2e_test/background_ddl/basic.slt' --junit "batch-ddl-${profile}"
```

Reply:
Ah, it appears I searched in an older branch somehow and found it in a pre-existing file `run-e2e-tests.sh`. I'll add it.